server.js

```diff
@@ -24,12 +17,141 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+// CREATE SCHEMA
+
+const { Schema } = mongoose;
+
+const songSchema = new Schema({
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+})
+
+const Song = mongoose.model("Song", songSchema);
+
+// RESET DB
+
+if (process.env.RESET_DB) {
+  const resetDatabase = async () => {
+    await Song.deleteMany();
+    topMusicData.forEach((singleSong) => {
+      const newSong = new Song(singleSong);
+      newSong.save()
+    })
+  }
+  resetDatabase();
+}
+
 // Start defining your routes here
+
+// ROUTE: START
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({
+    Message: "Here you can browse the top music data. Below you find the endpoints you can use 🎶",
+    Routes: [
+      {
+        "/songs": "= all songs",
+        "/songs/id/:id": "= a specific song based on id, write the id of the song instead of :id",
+        "/songs/artist/:artistname": "= get the titles from a specific artist, write the artist instead of :artistname"
+      }
+    ],
+  })
 });
 
-// Start the server
+
+
+
+// ROUTE: ALL SONGS
+
+app.get("/songs", async (req, res) => {
+  const songs = topMusicData;
+  if (songs) {
+    res.status(200).json({
+      success: true,
+      message: "OK",
+      body: {
+        topMusicData: songs
```

Comment:
Name it something other than topMusicData, it gets a bit confusing that it's called the same as the data json file. Maybe "content"?